docs/code-examples/transform3d_simple_v2.cpp

```diff
@@ -0,0 +1,31 @@
+// Log some transforms.
+
+#include <rerun.hpp>
+
+#include <cmath>
+
+namespace rr = rerun;
+namespace rrd = rr::datatypes;
+
+const float pi = static_cast<float>(M_PI);
+
+int main() {
+    auto rr_stream = rr::RecordingStream("transform3d");
+    rr_stream.connect("127.0.0.1:9876");
+
+    auto arrow = rr::archetypes::Arrows3D({0.0f, 1.0f, 0.0f});
```

Comment:
I think the api looks really nice but have one thought. In Python we expose archetypes in the top level namespace (module) as well as under rerun.archetypes. That makes it possible to write some simple logging code without introducing the concept of archetypes yet. Is that something we might want to do in C++ as well? Might be a bit unclean for C++, but it is consistent with Python and we definitely want it there. Perhaps a bit magic but we could potentially do it inside `rerun.hpp` since that's what you include when you want the simple stuff (although that might be a little confusing)

Reply:
> If we do it here we should do so in Rust as well :thinking: 

That's the plan yep. We don't do so yet because we were still "hiding" the new APIs until now (this wasn't a problem in python because we exposed those archetypes at the root of `rr2`, not `rr`).

The death of MsgSender will mark the beginning of archetypes at the root of the `rerun` crate.